pkg/pillar/flextimer/flextimer.go

```diff
@@ -1,31 +1,34 @@
-// Copyright (c) 2018 Zededa, Inc.
+// Copyright (c) 2018-2024 Zededa, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-// Provide randomized timers - botn based on range and binary exponential
+// Provide randomized timers - both based on range and binary exponential
 // backoff.
 // Usage:
 //  ticker := NewRangeTicker(min, max)
 //  select ticker.C
 //  ticker.UpdateRangeTicker(newmin, newmix)
 //  ticker.StopTicker()
 // Usage:
-//  ticker := NewExpTicker(start, max, randomFactor)
+//  ticker := NewExpTicker(start, max, noise)
 //  select ticker.C
-//  ticker.UpdateRangeTicker(newstart, newmax, newRandomFactor)
+//  ticker.UpdateExpTicker(newstart, newmax, newNoise)
 //  ticker.StopTicker()
 
 package flextimer
 
 import (
 	"math/rand"
 	"time"
+
+	"github.com/sirupsen/logrus"
 )
 
 // Take min, max, exp bool
-// If exp false then [min, max] is random range
+// If exp false then [min, max] is random range.
 // If exp true then start at min and do binary exponential backoff
-// until hitting max, then stay at max. Randomize +/- randomFactor
-// When config is all zeros, then stop and close channel
+// until hitting max, then stay at max. Randomize +/- noise, which
+// is in the same units as min and max. When config is all zeros,
+// then stop and close channel
```

Comment:
I'm surprised I didn't add a reference.
The rangeticker (and using the 0.3 factor with it) comes from the paper at https://ee.lbl.gov/papers/sync_94.pdf 
Might make sense to add that to the comments.

Note that I didn't have any basis for the random factor for the expticker; changing that to a noise time is fine with me (and it does make it easier to understand). Thanks for improving this.